adb-connect, adb-pair, adb-disconnect: add page (#15250)
tldr-pages/tldr: pages/common/adb-pair.md

# adb pair

> > This command is an alias of `adb connect`.
> 
> - View documentation for the original command:

Author: ```suggestion
> This command has been moved.

- View documentation for `adb pair:
```
And same here.

Author: Sounds reasonable
